ils_middleware/tasks/folio/mappings/bf_work.py

```diff
@@ -79,6 +79,14 @@
     <{bf_work}> bf:subject ?subject_node .
     OPTIONAL {{
         ?subject_node rdfs:label ?subject .
+
```

Comment:
The genre SPARQL query template should be it's own variable instead being a part of the `subject` variable otherwise it is an invalid SPARQL query, i.e.

```python
genre = """PREFIX bf: <http://id.loc.gov/ontologies/bibframe/>
PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>

SELECT ?genre
WHERE {{
    <{bf_work}> a bf:Work .
    <{bf_work}> bf:genreForm ?genre_node .
    OPTIONAL {{
        ?genre_node rdfs:label ?genre .
    }}
}}"""
```

You'll then need to add the new genre template to the following places:

- build.py's [transformers dict](=https://github.com/LD4P/ils-middleware/blob/daa4a75cbe41bfc08aed870d7d84b60c3e96c300/ils_middleware/tasks/folio/build.py#L245-L260), something like `genre: _subjects,`
- map.py's [BF_TO_FOLIO_MAP](https://github.com/LD4P/ils-middleware/blob/daa4a75cbe41bfc08aed870d7d84b60c3e96c300/ils_middleware/tasks/folio/map.py#L12)